src/ape/types/__init__.py

```diff
@@ -184,6 +183,54 @@ class ContractLog(BaseModel):
     Is `None` when from the pending block.
     """
 
+    data: Any = Field(default=None)
+    """The data hash of the event."""
+
+    topics: List = Field(default=[])
+    """The hashed topics of the event."""
+
+    @property
+    def event_name(self) -> str:
+        return self.abi.name
+
+    @cached_property
+    def event_arguments(self) -> Dict[str, Any]:
```

Comment:
I like this implementation. I wanted something like this but wasn't sure how to attack it. This is perfect